spec/unit/recipes/server_spec.rb

```diff
@@ -1,18 +1,20 @@
 require_relative '../../spec_helper'
 
 describe 'osl-acme::server' do
-  ALL_PLATFORMS.each do |p|
-    context "#{p[:platform]} #{p[:version]}" do
-      cached(:chef_run) do
-        ChefSpec::SoloRunner.new(p).converge(described_recipe)
-      end
-      before do
-        stub_command('/usr/local/go/bin/go version | grep "go1.5 "')
-        stub_command('screen -list boulder | /bin/grep 1\ Socket\ in')
-      end
-      it 'converges successfully' do
-        expect { chef_run }.to_not raise_error
-      end
+  p = CENTOS_7
+  context "#{p[:platform]} #{p[:version]}" do
+    cached(:chef_run) do
+      ChefSpec::SoloRunner.new(p).converge(described_recipe)
+    end
```

Comment:
Since we're only checking one platform I recommend you just remove this context and just put ``CENTOS_7``  in the ``ChefSpec::SoloRunner.new`` method call.